packages/dashboards/src/lib/components/timeseries-widget/chart-presets/xy-chart/xy-chart.component.html

```diff
@@ -4,28 +4,78 @@
+                        (click)="onLegendClick(legendSeries, $event)">
                 <nui-rich-legend-tile [value]="chartAssist.getHighlightedValue(legendSeries, 'y', undefined, valueAccessorKey)"
-                                      [backgroundColor]="chartAssist.palette.standardColors.get(legendSeries.id)"
-                                      [color]="chartAssist.palette.textColors.get(legendSeries.id)">
+                            [backgroundColor]="chartAssist.palette.standardColors.get(legendSeries.id)"
+                            [color]="chartAssist.palette.textColors.get(legendSeries.id)">
                     <nui-chart-marker rightEdge
-                                      [marker]="chartAssist.markers.get(legendSeries.id)"
-                                      [color]="chartAssist.palette.standardColors.get(legendSeries.id)">
+                                [marker]="chartAssist.markers.get(legendSeries.id)"
+                                [color]="chartAssist.palette.standardColors.get(legendSeries.id)">
                     </nui-chart-marker>
                 </nui-rich-legend-tile>
+
+                <div description
+                    class="legend-description-container nui-text-small d-flex flex-column justify-content-center px-2"
+                    [ngClass]="['state-' + chartAssist.renderStatesIndex[legendSeries.id]?.state]">
+
+                        State - {{chartAssist.renderStatesIndex[legendSeries.id]?.state}}
+                    <div *ngIf="isSeriesInteractive(legendSeries); else noLink"
+                        class="legend-description"
+                        (click)="$event.stopPropagation()">
+                        <div *ngIf="!legendSeries.link" 
+                            class="legend-description-primary link" 
+                            [title]="legendSeries.legendDescriptionPrimary"
+                            (click)="onInteraction(legendSeries, $event)">
+                                HANDLER - {{legendSeries.legendDescriptionPrimary}}
+                        </div>
+                        <a class="legend-description-primary link"
+                            *ngIf="legendSeries.link" 
+                            [href]="legendSeries.link" 
+                            rel="noopener noreferrer"
+                            [title]="legendSeries.legendDescriptionPrimary">
+                                HREF - {{legendSeries.legendDescriptionPrimary}}
+                        </a>
+                        <div (click)="onInteraction(legendSeries, $event)"
+                            class="legend-description-secondary"
+                            *ngIf="!legendSeries.secondaryLink" 
+                            [title]="legendSeries.legendDescriptionSecondary">
+                                {{legendSeries.legendDescriptionSecondary}}
+                        </div>
+                        <div *ngIf="legendSeries.secondaryLink">
```

Comment:
is this extra div necessary? I notice that the <a> tag on line 44 doesn't have a surrounding div.

Reply:
I am honestly not sure. But when it isn't surrounded by the div tag then it will not be displayed at all.